<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lb_store">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>店铺：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_showStoreID">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_memberID">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>91</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>会员ID：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_showMemID">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>141</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_phone">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>账号：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_token">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>80</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>token：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_showPhone">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>200</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_showToken">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>471</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_language">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>60</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>语言:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_showLanguage">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>60</y>
     <width>70</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>zh</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tW_showGoodsInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>531</width>
     <height>511</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>商品编号</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>商品名称</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>剩余数量</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>操作</string>
    </property>
   </column>
  </widget>
  <widget class="QTextEdit" name="tE_runningStat">
   <property name="geometry">
    <rect>
     <x>563</x>
     <y>240</y>
     <width>390</width>
     <height>460</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_retData">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>186</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>返回数据：</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_runNotice">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>10</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_getHomePage">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>获取主页</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_getGoodsList">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>10</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>获取商品列表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pBgetGoodsInfo">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>10</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>获取商品详情</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_showTime">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
